app/models/schedule.rb

```diff
@@ -11,6 +11,10 @@ def self.times
     [:morning, :afternoon, :evening]
   end
 
+  def self.correct_size
+    days.size * times.size
+  end
```

Comment:
I guess we could also convert these methods into constants (`DAYS`, `TIMES`, `CORRECT_SIZE`) to save a few milliseconds ;-)